src/components/FeatureViews/Courses/CourseManagement.js

```diff
@@ -0,0 +1,469 @@
+  };
+
+  const GET_COURSES = gql`
+  query getCourses {
+    courses {
+      dayOfWeek
+      endDate
+      endTime
+      title
+      startTime
+      startDate
+      instructor {
+        user {
+          firstName
+          lastName
+        }
+      }
+      courseId
+    }
+  }
+  `;
+
+  const { data, loading, error } = useQuery(GET_COURSES);
+
+  if (loading) return <Loading />;
+  if (error) return console.error(error.message);
+  // console.log(data)
+
+  const courseDisplayList = data.courses.map((courses) => (
+    <CourseDisplayCard
+      title={courses.title}
+      day={courses.dayOfWeek}
+      endDate={courses.endDate}
+      endTime={courses.endTime}
+      startTime={courses.startTime}
+      startDate={courses.startDate}
+      instructor={courses.instructor}
+      id={courses.courseId}
+    />
+  ));
+
+  return (
+    <Grid item xs={12}>
+      <BackgroundPaper elevation={2}>
+        <Typography align="left" className="heading" variant="h3">
+          Course Management
+        </Typography>
+        <Paper elevation={4} className={classes.appBar}>
+          <Grid
+            container
+            alignItems="center"
+            className={classes.containerMargins}
+          >
+            <Grid item xs={3}>
+              <FormControl className={classes.margin}>
+                <Select
+                  labelId="demo-customized-select-label"
+                  id="demo-customized-select"
+                  displayEmpty
+                  value={tab1}
+                  onChange={handleChange1}
+                  classes={{ select: classes.menuSelect }}
+                  input={<BootstrapInput />}
+                  MenuProps={{
+                    classes: { list: classes.dropdown },
+                    anchorOrigin: {
+                      vertical: "bottom",
+                      horizontal: "left",
+                    },
+                    transformOrigin: {
+                      vertical: "top",
+                      horizontal: "left",
+                    },
+                    getContentAnchorEl: null,
+                  }}
+                >
+                  {tab1 === "" ? <MenuItem value="">Sort By</MenuItem> : null}
+                  <MenuItem
+                    className={classes.menuSelect}
+                    value={"start-date"}
+                    ListItemClasses={{ selected: classes.menuSelected }}
+                  >
+                    Start Date (Latest)
+                  </MenuItem>
+                  <MenuItem
+                    className={classes.menuSelect}
+                    value={"class-name"}
+                    ListItemClasses={{ selected: classes.menuSelected }}
+                  >
+                    Class Name(A-Z)
+                  </MenuItem>
+                </Select>
+              </FormControl>
+            </Grid>
+            <Grid item xs={3}>
+              <FormControl className={classes.margin}>
+                <Select
```

Comment:
it looks like all of your Selects are using very similar code (all the same structure except the values are different. Try to make a higher order component here. It can be called something like `<CourseFilter>` or something, and it'd be something like...
```
const CourseFilter = ( { options } ) => {
  const classes = useStyles();
  const [ value, setValue ] = useState("");
  const handleChange = (event) => setValue(event.target.value);
  return (<Select
                    displayEmpty
                    value={value}
                    onChange={handleChange}
                    ...the rest of the props you have for this current Selects you're creating
                   >
                   { options.map( (option) => 
                            (<MenuItem
                                    className={classes.menuSelect}
                                    ...other menu item props
                              >
                              {option.value}
                        </MenuItem>))
                    }
    </Select>)
}
```
Then you'd just use this component a couple of times here in the `<CourseManagement>` component. You'll need to figure out a way to pass the value in this `<CourseFilter/>` component to the `<CourseManagement/>` component. You can either do this through props or through React Context.